modules/37_industry/subsectors/equations.gms

```diff
@@ -211,51 +221,42 @@ q37_chemicals_feedstocks_limit(t,regi) ..
   * p37_chemicals_feedstock_share(t,regi)
 ;
 
-*' Define the flow of non-energy feedstocks. It is used for emissions accounting and calculating plastics production
+*' Define the flow of non-energy feedstocks. It is used for emissions accounting
+*' and calculating plastics production
 q37_demFeFeedstockChemIndst(t,regi,entyFe,emiMkt)$(
                          entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt) ) ..
-  sum(se2fe(entySe,entyFe,te),
+  sum(sefe(entySe,entyFe),
     vm_demFENonEnergySector(t,regi,entySe,entyFe,"indst",emiMkt)
   )
   =e=
-  sum((fe2ppfEn(entyFe,ppfen_industry_dyn37(in)),
-       secInd37_emiMkt(secInd37,emiMkt),
-       secInd37_2_pf(secInd37,in_chemicals_feedstock_37(in))),
-    ( vm_cesIO(t,regi,in)
-    + pm_cesdata(t,regi,in,"offset_quantity")
+    sum((sefe(entySe,entyFe),
+         fe2ppfEn(entyFe,in),
+         in_chemicals_feedstock_37(in),
+         secInd37_2_pf(secInd37,in),
+         secInd37_emiMkt(secInd37,emiMkt)),
+      v37_demFeIndst(t,regi,entySe,entyFe,in,emiMkt)
     )
   * p37_chemicals_feedstock_share(t,regi)
-  )
 ;
 
 *' Feedstocks flow has to be lower than total energy flow into the industry
-q37_feedstocksLimit(t,regi,entySe,entyFe,emiMkt)$(
-                                             sefe(entySe,entyFe)
-                                         AND sector2emiMkt("indst",emiMkt)
-                                         AND entyFe2Sector(entyFe,"indst")
-                                         AND entyFeCC37(entyFe)            ) ..
-  vm_demFeSector(t,regi,entySe,entyFe,"indst",emiMkt)
-  =g=
-  vm_demFENonEnergySector(t,regi,entySe,entyFe,"indst",emiMkt)
-;
-
-*' Feedstocks have identical fossil/biomass/synfuel shares as industry FE
-q37_feedstocksShares(t,regi,entySe,entyFe,emiMkt)$(
-                         sum(te, se2fe(entySe,entyFe,te))
-                     AND entyFE2sector2emiMkt_NonEn(entyFe,"indst",emiMkt)
-                     AND cm_emiscen ne 1                                   ) ..                 
-    vm_demFeSector_afterTax(t,regi,entySe,entyFe,"indst",emiMkt)
-  * sum(se2fe(entySe2,entyFe,te),
-      vm_demFENonEnergySector(t,regi,entySe2,entyFe,"indst",emiMkt)
-    )
-  =e=
+q37_feedstocksLimit(t,regi,entySe,entyFe,in,secInd37,emiMkt)$(
```

Comment:
do we need this restriction with the new formulation of  `q37_demFeFeedstockChemIndst`?

Reply:
Which one?  The rats-nest of lines 244-252?  Actually, yes.  This can be cleaned up by introducing some specialised mappings for the purpose, but the beauty treatment is scheduled for next week `;)`